include/OpenColorIO/OpenColorIO.h

```diff
@@ -572,6 +572,43 @@ class OCIOEXPORT Config
      */
     void setInactiveColorSpaces(const char * inactiveColorSpaces);
     const char * getInactiveColorSpaces() const;
+    
+    /**
+     * \brief Return true if the specified color space is linear.
```

Comment:
Please ignore the isColorSpaceLinear stuff.  We built the PR on that branch since we needed the functionality.